chains/solana/contracts/programs/ccip-offramp/src/event.rs

```diff
@@ -5,9 +5,7 @@ use crate::state::MessageExecutionState;
 
 #[event]
 pub struct CommitReportAccepted {
-    // Will be zero'd (MerkleRoot::Default() in case the commit report included no
-    // merkle root. This is to circumvent a Go deserialization issue.)
-    pub merkle_root: MerkleRoot,
+    pub merkle_root: Option<MerkleRoot>,
```

Comment:
We need to communicate this change with the following teams:
- Chain Reader
- Log Poller
- Observability

Reply:
I've requested this change to fix decoding the report in CR/offchain so LGTM 👍🏻